collection_requirements.rst

```diff
@@ -0,0 +1,114 @@
+*****************************
+Ansible Collections Checklist
+*****************************
+
+.. contents:: Topics
+
+Overview
+========
+
+**Warning:** Subject to frequent updates
+       This is a "living document", expect it to change as we progress with the Collections work over the next few months.
+
+Feedback
+========
+
+As with any project it's very important that we get feedback from users, contributors and maintainers. We recognize that the move to Collections is a big change in how Ansible is developed and delivered.
+
+Please raise feedback by
+
+* Discussing in ``#ansible-community`` in Freenode IRC
+* Adding to the `Community Working Group IRC meeting <https://github.com/ansible/community/issues/539>`_
+* Creating `GitHub Issues <https://github.com/ansible-collections/overview/issues>`_ against this repo
+
+
+Why is this needed
+===================
+
+
+Repo structure
+===============
+
+galaxy.yml
+----------
+
+* tags MUST set
+* dependencies MUST be set to ``'>=1.0.0'`` (Might have to think about dependency order when publishing)
+
+meta/runtime.yml
+----------------
+Example: `meta/runtime.yml <https://github.com/ansible-collections/collection_template/blob/main/meta/runtime.yml>`_
+* MUST define the minimim version of Ansible which this collection works with
+  * If the Collection works with Ansible 2.9, then this shuld be set to
+
+Modules & Plugins
+------------------
+
+Documentation
+~~~~~~~~~~~~~~
+All module and plugin ``DOCUMENTATION`` MUST
+
+* Use the FQCN for ``M(...)`` FIXME Docs link
+* Use the FQCN for ``SEEALSO`` FIXME Docs link
+* Use the FQCN for ``extends_documentation_fragment:``, unles you are referring to doc_fragments from ansible-base
+
+Contributor Workflow
+====================
+
+Changelogs
+~~~~~~~~~~
+
+FIXME: What's the actual minimum requirement here, is it a file called X?
```

Comment:
Preferred (in descending order):
1. Use antsibull-changelog;
2. Provide `changelogs/changelog.yaml` in the [correct format](https://github.com/ansible-community/antsibull-changelog/blob/main/docs/changelog.yaml-format.md);
3. Provide a link to the changelog file (self-hosted).

The big question is how that link should be provided. We probably need to create a small DB somewhere with that info.